translations/de.log

```diff
@@ -0,0 +1,97 @@
+*****: de
+orig->languageName: 'en', 'de'
+orig->control: 'Control', 'Steuerung'
+orig->print: 'Print', 'Druck'
+orig->status: 'Status', 'Status'
+orig->console: 'Console', 'Konsole'
+orig->setup: 'Setup', 'Setup'
+orig->current: 'Current°C', 'Istwert°C'
+orig->active: 'Active°C', 'Aktiv°C'
+orig->standby: 'Standby°C', 'Standby°C'
+orig->move: 'Move', 'Bewegung'
+orig->extrusion: 'Extrusion', 'Extrusion'
+orig->macro: 'Macro', 'Makro'
+orig->stop: 'STOP', 'STOP'
+orig->extruderPercent: 'Extruder%', 'Extruder%'
+orig->speed: 'Speed ', 'Tempo '
+orig->fan: 'Fan ', 'Lüfter '
+orig->timeRemaining: 'Time left: ', 'Restzeit: '
+orig->simulated: 'sim'd ', 'Datei '
+orig->file: 'file ', 'Simul. '
+orig->filament: 'filament ', 'Filament '
+orig->slicer: 'slicer ', 'Slicer '
+orig->notAvailable: 'n/a', 'n/v'
+orig->pause: 'Pause', 'Pause'
+orig->babystep: 'Baby step', 'Einzelschritt'
+orig->resume: 'Resume', 'Fortsetzen'
+orig->cancel: 'Cancel', 'Abbrechen'
+orig->reprint: 'Print again', 'Erneut drucken'
+orig->resimulate: 'Simulate again', 'Erneut simulieren'
+orig->set: 'Set', 'Set'
+orig->volume: 'Volume ', 'Lautstärke '
+orig->calibrateTouch: 'Calibrate touch', 'Touch kalibrieren'
+orig->mirrorDisplay: 'Mirror display', 'Anzeige spiegeln'
+orig->invertDisplay: 'Invert display', 'Anzeige umkehren'
+orig->theme: 'Theme', 'Darstellung'
+orig->brightnessDown: 'Brightness -', 'Beleuchtung  -'
+orig->brightnessUp: 'Brightness +', 'Beleuchtung  +'
+orig->saveSettings: 'Save settings', 'Einstllgen sichern'
+orig->clearSettings: 'Clear settings', 'Werks-Reset'
+orig->saveAndRestart: 'Save & Restart', 'Sichern & Reboot'
+orig->infoTimeout: 'Info timeout ', 'Info timeout '
+orig->screensaverAfter: 'Screensaver ', 'Screensaver '
```

Comment:
"Bildschirmschoner" 
If it is to long, leave "Screensaver"

Reply:
Done.